modules/loader/src/test/scala/com/snowplowanalytics/snowplow/rdbloader/test/PureDAO.scala

```diff
@@ -80,9 +80,6 @@ object PureDAO {
 
     def executeQueryList[A](query: Statement)(implicit A: Read[A]): Pure[List[A]] =
       results.executeQuery.asInstanceOf[Statement => Pure[List[A]]](query)
-
-    def executeQueryOption[A](query: Statement)(implicit A: Read[A]): Pure[Option[A]] =
```

Comment:
It also looks like to make the tests pass we have to identify the all `case Statement.ManifestGet(_)` occurrences and replace Some/None with Lists :)